src/Statistics.jl

```diff
@@ -227,34 +231,63 @@ function std(
 )
     return sqrt(var(M, x, m; corrected = corrected, kwargs...))
 end
+
 @doc doc"""
-    m,v = mean_and_var(M,x,w)
+    mean_and_var(M, x, w; corrected = false, kwargs...) -> (mean, var)
+    mean_and_var(M, x; corrected = true, kwargs...) -> (mean, var)
 
 compute the [`mean`](@ref) `m` and the [`var`](@ref)iance `v` simultaneously.
 """
 function mean_and_var(
     M::Manifold,
     x::AbstractVector,
-    w::AbstractWeights = Weights(ones(length(x)), length(x));
+    w::AbstractWeights;
+    corrected = false,
     kwargs...
 )
     m = mean(M, x, w; kwargs...)
-    v = var(M, x, w, m; kwargs...)
+    v = var(M, x, w, m; corrected = corrected, kwargs...)
     return m, v
 end
+
+function mean_and_var(
```

Comment:
Can't this function be replaced with setting `w::AbstractWeights = ProbabilityWeights(ones(n), n)` in line 244 as it was before? That seems to be the only difference between both versions.

Reply:
The other difference is the change in `corrected`. Unweighted `var` has `corrected=true` and weighted `var` has `corrected=false`.

Reply:
Ah I did not see that, sorry; then tis is fine, too.